scripts/test.sh

```diff
@@ -2,22 +2,22 @@
 
 set -e
 
-export PATH=env/bin:${PATH}
+export PATH=$(poetry show -v | grep virtualenv | sed 's/Using virtualenv: //'):${PATH}
```

Comment:
It seems this path resolves to that of the virtualenv, rather than of its `bin/` directory. I think you'll need to add `/bin/ ` to the end of all of these.

Reply:
You are correct!

Reply:
8a6b01b